app/src/main/java/com/android/sample/model/recipe/RecipesRepository.kt

```diff
@@ -1,78 +1,9 @@
 package com.android.sample.model.recipe
 
-/** Interface for retrieving recipe data. */
```

Comment:
1. **Inheritance Change:**  
   Smart move making `RecipesRepository` inherit from `RecipeNetworkRepository`—this really simplifies things and removes unnecessary duplication. Nice job cleaning up the interface!

2. **Deleted Methods:**  
   Makes total sense to remove those methods since they’re already handled by `RecipeNetworkRepository`. Less code duplication = easier maintenance. Just ensure no functionality was inadvertently affected.

Reply:
Hey I think it should be ok. I ran all the tests and everything is good! 😃